instrumentation-api/src/test/java/io/opentelemetry/instrumentation/api/instrumenter/InstrumenterTest.java

```diff
@@ -196,8 +187,8 @@ void server() {
                         span.hasName("span")
                             .hasKind(SpanKind.SERVER)
                             .hasInstrumentationScopeInfo(InstrumentationScopeInfo.create("test"))
-                            .hasTraceId(spanContext.getTraceId())
-                            .hasSpanId(spanContext.getSpanId())
+                            .hasTraceId(Span.fromContext(context).getSpanContext().getTraceId())
+                            .hasSpanId(Span.fromContext(context).getSpanContext().getSpanId())
```

Comment:
just curious if there was a particular reason to inline `spanContext` here?

Reply:
Hm, don't think so, I probably just inlined it automatically without thinking. I'll revert that.